ToNote/Views/MainView.xaml.cs

```diff
@@ -1,7 +1,12 @@
 ﻿namespace ToNote
 {
     using System;
+    using System.Collections;
```

Comment:
Would be nice if you cleaned up the unused usings. Would be even nicer if you used VS autoformatters and removed unused using solution-wise, as well as formatted the code (ctrl+K, ctrl+D for document, ctrl+K,ctrl+F for selected section by default)